internal/util/cephcmds.go

```diff
@@ -249,27 +249,27 @@ func CreateObject(ctx context.Context, monitors string, cr *Credentials, poolNam
 // RemoveObject removes the entire omap name passed in and returns ErrObjectNotFound is provided omap
 // is not found in rados
 func RemoveObject(ctx context.Context, monitors string, cr *Credentials, poolName, namespace, oMapName string) error {
-	// Command: "rados <options> rm oMapName"
-	args := []string{
-		"-m", monitors,
-		"--id", cr.ID,
-		"--keyfile=" + cr.KeyFile,
-		"-c", CephConfigPath,
-		"-p", poolName,
-		"rm", oMapName,
+	conn, err := connPool.Get(monitors, cr.ID, cr.KeyFile)
+	if err != nil {
+		return err
+	}
+	defer connPool.Put(conn)
+
+	ioctx, err := conn.OpenIOContext(poolName)
+	if err != nil {
```

Comment:
if the pool is not present, we may need to return the `ErrObjectNotFound`

Reply:
In case `ErrPoolNotFound` is returned, it will not get translated to `ErrObjectNotFound`.